admin/app/components/solidus_admin/shipping_methods/form/component.html.erb

```diff
@@ -0,0 +1,137 @@
+<%= form_for @shipping_method, url: @url, html: { id: @id } do |f| %>
+  <%= page_with_sidebar do %>
+    <%= page_with_sidebar_main do %>
+      <%= render component("ui/panel").new(title: t(".details")) do |panel| %>
+        <% panel.with_section(class: "flex flex-col gap-4") do %>
+          <div class="flex gap-4 w-full">
+            <%= render component("ui/forms/field").text_field(f, :name) %>
+            <%= render component("ui/forms/field").text_field(f, :admin_name) %>
+          </div>
+          <div class="flex gap-4 w-full">
+            <%= render component("ui/forms/field").text_field(f, :code) %>
+            <%= render component("ui/forms/field").text_field(f, :carrier) %>
+            <%= render component("ui/forms/field").text_field(f, :service_level) %>
+          </div>
+          <%= render component("ui/forms/field").text_field(
+            f,
+            :tracking_url,
+            placeholder: t('spree.tracking_url_placeholder')
+          ) %>
+        <% end %>
+      <% end %>
+
+      <%= render component("ui/panel").new(title: t('.pricing')) do |panel| %>
+        <% panel.with_section do %>
+          <%= render component("calculators/form").new(
+            form: f,
+            calculators: @calculators
+          ) %>
+        <% end %>
+      <% end %>
+
+      <%= render component("ui/panel").new(title: t(".taxation")) do |panel| %>
+        <% panel.with_section do %>
+          <%= render component("ui/forms/field").select(
+            f.object_name,
+            :tax_category_id,
+            Spree::TaxCategory.all.map { |tc| [tc.name, tc.id] },
+            object: @shipping_method,
+            value: @shipping_method.tax_category_id,
+            class: "fullwidth"
+          ) %>
+        <% end %>
+      <% end %>
+    <% end %>
+
+    <%= page_with_sidebar_aside do %>
+      <%= render component("ui/panel").new(title: t('.availability')) do |panel| %>
+        <% panel.with_section do %>
+          <div class="flex flex-col gap-4">
+            <%= render component("ui/forms/field").select(
+              f.object_name,
+              :store_ids,
+              Spree::Store.all.map { |tc| [tc.name, tc.id] },
+              object: @shipping_method,
+              value: @shipping_method.store_ids,
+              class: "fullwidth",
+              multiple: true,
+              label: t(".stores")
+            ) %>
+
+            <%= render component("ui/forms/field").select(
+              f.object_name,
+              :zone_ids,
+              Spree::Zone.all.map { |tc| [tc.name, tc.id] },
+              object: @shipping_method,
+              value: @shipping_method.zone_ids,
+              class: "fullwidth",
+              multiple: true,
+              name: "#{f.object_name}[zones][]",
+              label: t(".zones")
+            ) %>
```

Comment:
The style seems broken for these two:

<img width="396" alt="image" src="https://github.com/user-attachments/assets/0fe2e789-6831-4e53-8ad3-737e5c743934" />

Checked both chrome an safari

Reply:
Well the select component is also broken on main for me, so I guess it's out of scope